site/_posts/2021-06-24-mitigate-acquisition-risks-auth-vendor-excerpt.md

```diff
@@ -0,0 +1,62 @@
+
+It's time to talk to your account manager about your business relationship. Give them a call or an email and try to negotiate a long-term contract that will protect your business interests and guarantee you some stability. The research into the contract you did previously? Now it the time to reference it.
+
+Don't forget to ask about migration timelines while you're at it, so you know how soon you need to be prepared for what transitions.
+
+If you don't have an account manager, send an email to the sales or support team. They may send you elsewhere, but are a good starting point.
+
+## Evaluate What It Would Take to Switch Vendors
+ 
+Budget some dev team time to look into other options to see what could be a better alternative in case it becomes necessary to move. Discuss the possibility with your partners or stakeholders, touching base with [everyone you discussed the issue with during the initial decision process](/learn/expert-advice/identity-basics/outsource-auth-system-blueprint/), so that everyone understands what it would take to make the switch. 
+
+Even if you stick with your vendor through the acquisition, at least now you know more, and you're prepared for whatever comes afterward. 
+
+## Consider Impact to Current or Planned Projects
+
+There's never a perfect time for a huge change to your auth system. What current projects will be impacted, for better or worse? Do you have projects in the planning stage that will have to be reimagined due to new standards or a different set of features? 
+
+It's best to discuss this with your stakeholders, again so that everyone is on the same page and has consensus about priorities.
+
+## Consider Other Options
+
+If the changes are disruptive enough, you may decide you don't want to use third-party solutions anymore. You have a few other options:
+
+* **Use a non SaaS solution:** SaaS solutions are great, but if you use a non-SaaS solution, where you host it yourself, you have far more control over any changes to functionality. You may have to upgrade for security or contractual reasons, but you'll be able to do it on your timeline, not the acquirer's.
+* **An open-source solution:** You'll still have to manage your own source code. You'll just be using a free and community-driven solution like [Gluu](gluu.org), [Keycloak](https://www.keycloak.org/index.html), or [OpenIAM](https://www.openiam.com/). Your team will have to [explore these projects](/learn/expert-advice/identity-basics/open-source-vs-commercial/) and choose the one that works best for your system.
+* **In-house custom build solution:** In some rare cases, none of the available solutions will be a good fit for your organization, and you'll decide to build a custom auth system for your product. This will require more resources to achieve but of course, when completed, you'll have a solution that works best for you and can be fine-tuned with more features in the future at your will—something you won't get anywhere else.
```

Comment:
long sentence - consider
you'll have a solution that works best for you.  Further,  it can be fine-tuned with more features in the future at your will—something you won't get anywhere else.